app/scripts/components/common/map/style-generators/raster-paint-layer.tsx

```diff
@@ -52,10 +52,34 @@ export function RasterPaintLayer(props: RasterPaintLayerProps) {
 
   useEffect(
     () => {
-      const tileParamsAsString = qs.stringify(updatedTileParams, {
+      // Create a modified version of the parameters
+      const processedParams = { ...updatedTileParams } as {
+        reScale?: number[];
+        colormap_name?: string;
+        bands?: string[];
+        assets?: string[];
```

Comment:
For my information - are there any more parameters that should be formatted repeatedly?   I learned that `bidx` also should be handled in that way from the recent bug 😬

Reply:
I am not exactly sure, will see if I can scrape the api docs for the parameters that require an array.